MonoGame.Framework/Platform/SDL/SDL2.cs

```diff
@@ -646,12 +646,20 @@ public struct ButtonEvent
             public uint TimeStamp;
             public uint WindowId;
             public uint Which;
-            public Button Button;
+            private byte _button;
             public ButtonState State;
             public byte Clicks;
             private byte _padding1;
             public int X;
             public int Y;
+
+            public Button Button
+            {
+                get
+                {
+                    return (Button)(1 << _button);
```

Comment:
Ha! Nice! A much better fix, only it's `1 << (_button - 1)`

Reply:
Oh.